apps/judicial-system/web/src/routes/Court/Indictments/Advocates/Advocates.tsx

```
@@ -59,6 +59,24 @@ const Advocates = () => {
             message={formatMessage(strings.alertBannerText)}
             type="info"
           />
+          {workingCase.defendants
+            ?.filter((defendant) => defendant.requestedDefenderChoice)
+            .map((defendant) => (
+              <Box marginTop={2}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)